oci/kafka/image.yaml

```diff
@@ -2,10 +2,10 @@ version: 1
     
 upload:  
   - source: "canonical/kafka-rock"
-    commit: 8811f9d9dff3703b954cf4b1c42fff8253abe15a
+    commit: 728311b40b55e13ada3112e10a59cd3aaabab048
     directory: .
     release:
-      3.6-22.04:
-        end-of-life: "2024-09-01T00:00:00Z"
+      3.9.1-22.04:
+        end-of-life: "2027-02-09T00:00:00Z"
```

Comment:
is this version LTS? Is your team actively maintaining it until 2027? If you have to patch it, aren't you going to bump the patch version?

Reply:
@marcoppenheimer @zhijie-yang